packages/rocketchat-lib/server/functions/deleteMessage.js

```diff
@@ -25,7 +25,7 @@ RocketChat.deleteMessage = function(message, user) {
 		}
 
 		Meteor.defer(function() {
-			RocketChat.callbacks.run('afterDeleteMessage', deletedMsg);
+			RocketChat.callbacks.run('afterDeleteMessage', deletedMsg || { _id: message._id });
```

Comment:
@sampaiodiego should this be call to the callbacks be ran outside the `if (keepHistory)`? Because if that setting is true, the message is being deleted to the user's perspective even though we aren't technically deleting it.

Reply:
right.. since the message will be suppressed from any endpoint, we should call the callback. I'll change it to reflect that.